### ISSUE TITLE
Add some sort of background

### ISSUE THREAD

**Issue by Jontpan** [AxlLind/2D-platformer#111]
Right now the background looks boring, something should be done.

**Comment by AxlLind** [comment]
Yeah, maybe. I don't want to lose the minimalistic aesthetic of the game though. Any idea's of a good background?

The difficulty is making it look good when you jump arbitrarily high.

**Comment by Jontpan** [comment]
Hmm, I don't have anything in mind, I could ask my artist friend if he wants to do something :) He is building his portfolio so I don't think he would mind.
